web/src/ChatBox.js

```diff
@@ -145,6 +145,23 @@ class ChatBox extends React.Component {
     return message.html;
   };
 
+  copyMessage(message) {
+    const tempElement = document.createElement("div");
+    tempElement.innerHTML = message;
```

Comment:
Use util function for copy, either in this repo or Casdoor

Reply:
I found the util function is Casdoor(Setting.js), and the code do not work here:
the code is like this :
```
export function getClickable(text) {
  return (
    <a onClick={() => {
      copy(text);
      showMessage("success", "Copied to clipboard");
    }}>
      {text}
    </a>
  );
}
```
and I think write a new function to Parse HTML into clipboard is better;
new function is like this : 
```
  copyMessageFromHTML(message) {
    const tempElement = document.createElement("div");
    tempElement.innerHTML = message;
    const text = tempElement.innerText;
    copy(text);
    Setting.showMessage("success", "Message copied successfully");
  }
```